### DISCUSSION TITLE
[DOC] useStateAction is not documented

### DISCUSSION THREAD

**Discussion by eduvhc** [next-safe-action/next-safe-action#121] [Ideas]
The [`useStateAction`](https://github.com/TheEdoRan/next-safe-action/blob/next/packages/next-safe-action/src/hooks.ts#L277) hook is not present in the documentation

https://next-safe-action.dev/docs/execution/hooks/usestateaction

**Comment by TheEdoRan** [comment]
v7 documentation is currently on the [next docs website](https://next.next-safe-action.dev/docs/execution/hooks/usestateaction), since it's the beta version.
